docker/README.md

```diff
@@ -5,11 +5,29 @@ The image is built from a versioned Rocker image, with additional R packages and
 It can be built with the following command run in this directory:
 
 ```
-docker buildx build . -t scpca-tools --platform linux/amd64
+docker buildx build . -t scpcatools --platform linux/amd64
 ```
 
-Note that this image does not include RStudio, in attempt to make the image smaller. 
+Note that this image does not include RStudio, in attempt to make the image smaller.
 
 This image is built automatically on every update to `scpcaTools` (`main`) and pushed to `ghcr.io/alexslemonade/scpca-tools:edge`.
 
-Release versions of `scpcaTools` will be tagged as `ghcr.io/alexslemonade/scpca-tools:latest` and tagged by version number.
+Release versions of `scpcaTools` will be tagged as `ghcr.io/alexslemonade/scpcatools:latest` and tagged by version number.
+
+The main image includes all of the recommended packages for `scpcaTools` (aside from those only required for development), as well as the Python `scanpy` and `scvi-tools` packages and dependencies.
+
+There are also smaller images that can be built, which include only the R packages required for `scpcaTools`, with particular additions for specific purposes. These include:
+
+- `scpcatools_slim.Dockerfile`: A smaller image that includes only the required R packages for `scpcaTools`, with no additional packages.
+- `scpcatools_anndata.Dockerfile`: The slim package with the addition of the `zellkonverter` package for reading and writing `AnnData` objects, and the Python `anndata` package.
+
+These images are built automatically on every update to `scpcaTools` (`main`) and pushed to `ghcr.io/alexslemonade/scpcatools:edge-slim` and `ghcr.io/alexslemonade/scpcatools:edge-anndata`, respectively.
+
+## Setting R and Python dependencies
+
+Dependency lock files are built using the `make-requirements.sh` script.
```

Comment:
This means we can't use `renv::snapshot()` to capture dependencies in R? We have to run this script for every new package that gets added? 

Another question I have is about how we maintain the separate images. How do we specify what is included in the "full" images vs. the "slim" images?

Reply:
Okay going through this more, it looks like we still use `renv::snapshot()` to create the initial `renv.lock`, then we run `make_requirements.sh` to update the other lock files? I think this documentation could use some clarification on the steps to add in new packages.

Reply:
Yes, you are correct that the main `renv.lock` is still created with `renv::snapshot()`, though we could use this script to update it as well. I didn't implement that, but it might be a good way to go.

Reply:
Okay, I'm fine if the docs get updated in a later PR, I just want to make sure I understand how things are maintained here. Based on my understanding everytime you add a new package would require two steps: 

1. Run `renv::snapshot()` while in the project, updating `renv.lock`. 
2. Run `make_requirements.sh` to update all the remaining lock files. 

For Python, we have to add to `requirements.in` and then run `make_requirements.sh`. 
Let me know if this is incorrect?

Reply:
Oh, I didn't necessarily mean in a later PR, just that I recognized they weren't really done yet!

Your two steps seem correct to me, I think I wanted to leave the initial `renv::snapshot()` as manual so that the inital changes would be more manual. All of the lockfiles generated from this script should be a subset of that main lockfile.  I think I can add a check so this script fails if the lockfile is not up to date...

Reply:
Yes I'm fine with keeping the initial update to the main lock file as manual!